### ISSUE TITLE
multi-scale segmentation

### ISSUE THREAD

**Issue by soroushr** [matterport/Mask_RCNN#1391]
The objects I am trying to segment substantially vary in size. Some could be 8x8 pixels and some could be 500x500 pixels, in a 700x700 image.

So far Mask-RCNN does a very good job detecting small objects, but it completely misses the large ones. I know that `RPN_ANCHOR_SCALES` and FPNs are supposed to take care of multi-scale issues. But no matter how much I tweak the `RPN_ANCHOR_SCALES` I cannot get larger objects.

Any insights?
Thanks!

**Comment by PhanDuc** [comment]
Try to change the `MASK_SHAPE` and `MINI_MASK_SHAPE`

**Comment by soroushr** [comment]
Thanks for your response @PhanDuc .
Aren't those parameters just for memory/storage purposes? According to Waleed's [blog](https://engineering.matterport.com/splash-of-color-instance-segmentation-with-mask-r-cnn-and-tensorflow-7c761e238b46) under section
**4. Segmentation Masks**:
> The small mask size helps keep the mask branch light. During training, we scale down the ground-truth masks to 28x28 to compute the loss, and during inferencing we scale up the predicted masks to the size of the ROI bounding box and that gives us the final masks, one per object.

I will give it a shot - have you experimented with it?

**Comment by PhanDuc** [comment]
It is better that you should check the Ipython notebook, where they showed how the model detected and performed classification. So you can change the params!
